<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>261</width>
      <height>201</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLineEdit" name="InputB_1">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="LabelA_1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
     <widget class="QLabel" name="LabelC_1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>A+B</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ResultC_1">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="LabelB_1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Calculate_1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>140</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Addiere</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="InputA_1">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLineEdit" name="InputB_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="LabelA_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
     <widget class="QLabel" name="LabelC_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>A-B</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ResultC_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="LabelB_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Calculate_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>140</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Subtrahiere</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="InputA_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>275</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain">
    <property name="title">
     <string>Main</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuChoose">
    <property name="title">
     <string>Choose</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionSub"/>
   </widget>
   <addaction name="menuMain"/>
   <addaction name="menuChoose"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAddieren">
   <property name="text">
    <string>Addieren</string>
   </property>
  </action>
  <action name="actionSubtrahieren">
   <property name="text">
    <string>Subtrahieren</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionSub">
   <property name="text">
    <string>Sub</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
